hw/ip/otbn/dv/model/otbn_model.cc

```diff
@@ -263,14 +263,14 @@ int OtbnModel::step(svLogic edn_rnd_data_valid,
   if (!iss)
     return -1;
 
-  assert(!is_xz(edn_rnd_data_valid));
+  assert(!is_xz(edn_rnd_cdc_done));
   assert(!is_xz(edn_urnd_data_valid));
 
   try {
-    if (edn_rnd_data_valid) {
-      uint32_t int_edn_rnd_data[8];
-      set_rnd_data(int_edn_rnd_data, edn_rnd_data);
-      iss->edn_rnd_data(int_edn_rnd_data);
+    // If EDN response interface sends an acknowledgment or RTL signals CDC done
+    // ISS forwards the 32b package and the CDC done signal.
+    if (edn_rnd_data_ack ^ edn_rnd_cdc_done) {
```

Comment:
I know I've just written this in another comment, but don't want to lose track: I think it's better to keep the `||` that you had before, but add an assertion saying that both shouldn't happen at once.

Also, thinking about it, aren't the two signals in different clock domains? I think that `edn_rnd_data_ack` and `edn_rnd_data` are on the EDN clock and `edn_rnd_cdc_done` is on the main clock (which is kind of the point of that signal!). I think this should look like a separate `always @(posedge clk_edn_i) ...` block in the SystemVerilog that calls some new `otbn_model_edn_step()` DPI function to handle stuff in the EDN clock domain.